src/tiles/grid/item-tile.ts

```diff
@@ -179,37 +202,36 @@ export class ItemTile extends LitElement {
       }
 
       .created-by,
-      .date-sorted-by {
+      .date-sorted-by,
+      .volume-issue {
         display: flex;
-        justify-content: center;
+        justify-content: left;
         align-items: flex-end; /* Important to start text from bottom */
-        height: 3rem;
-        padding-top: 1rem;
+        padding: 10px 5px 5px 5px;
       }
 
       .truncated {
         flex: 1;
         color: #2c2c2c;
         min-width: 0; /* Important for long words! */
-        text-align: center;
-        line-height: 2rem;
+        text-align: left;
+        line-height: 15px;
         text-overflow: ellipsis;
         overflow: hidden;
         word-wrap: break-word;
-        -webkit-line-clamp: 2;
+        -webkit-line-clamp: 3;
         -webkit-box-orient: vertical;
+        padding-bottom: 1.5px;
       }
 
-      .truncated span {
+      span {
         font-size: 1.4rem;
         display: -webkit-box;
       }
 
       h1.truncated {
-        margin-top: 0rem;
-        margin-bottom: 0.5rem;
+        margin: 0px;
         font-size: 1.6rem;
```

Comment:
The ticket description says 14px for title and description.

Reply:
nice catch, will update it